tests/fv3_conf/cpld_control_run.IN

```diff
@@ -50,6 +50,12 @@ cp    @[INPUTDATA_ROOT]/CICE_FIX/@[OCNRES]/grid_cice_NEMS_mx@[OCNRES].nc .
 cp    @[INPUTDATA_ROOT]/CICE_FIX/@[OCNRES]/kmtu_cice_NEMS_mx@[OCNRES].nc .
 cp    @[INPUTDATA_ROOT]/CICE_FIX/@[OCNRES]/mesh.mx@[OCNRES].nc .
 
+# WW3 fix/input
+if [[ $CPLWAV == .T. && $CPLWAV2ATM == .T. ]]; then
```

Comment:
We would need this only if CPLWAV is true,  cplwave2atm=f would still need mod_defs and the input file